engine/runtime-instrument-repl-debugger/src/main/java/org/enso/interpreter/instrument/ReplDebuggerInstrument.java

```diff
@@ -85,7 +85,8 @@ protected OptionDescriptors getOptionDescriptors() {
   }
 
   /** The actual node that's installed as a probe on any node the instrument was launched for. */
-  public static class ReplExecutionEventNode extends ExecutionEventNode {
+  public static class ReplExecutionEventNodeImpl extends ExecutionEventNode
```

Comment:
Probably this class no longer needs to be `public` when it implements a public interface.

Reply:
Sure, yeah